src/quo2/components/list_items/account_list_card/view.cljs

```diff
@@ -0,0 +1,36 @@
+(ns quo2.components.list-items.account-list-card.view
+  (:require
+    [quo2.components.avatars.account-avatar.view :as account-avatar]
+    [quo2.components.icon :as icon]
+    [quo2.components.markdown.text :as text]
+    [quo2.foundations.colors :as colors]
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [quo2.components.list-items.account-list-card.style :as style]))
+
+(defn- internal-view
+  [{:keys [theme account-props network state action]}]
+  [rn/view
+   {:style               (style/container state theme)
+    :accessibility-label :container}
+   [rn/view
+    {:style {:flex-direction :row
+             :align-items    :center}}
+    [account-avatar/view account-props]
+    [rn/view {:style {:margin-left 8}}
+     [text/text
+      {:weight :semi-bold
+       :size   :paragraph-2} (:name account-props)]
+     [text/text
+      {:size  :paragraph-2
+       :style {:color (get colors/networks network)}} "eth:"
+      [text/text
+       {:size  :paragraph-2
+        :style {:color (colors/theme-colors colors/neutral-50 colors/neutral-40 theme)}}
+       (:address account-props)]]]]
+   (when (= action :icon)
+     [icon/icon :i/options
+      {:color               (colors/theme-colors colors/neutral-50 colors/neutral-40 theme)
+       :accessibility-label :icon}])])
```

Comment:
Is there any possibility this icon may change? If yes, we can get the icon name in props.

Reply:
Nope it wouldn't change

Reply:
@OmarBasem - Quick question: Do we need an `on-press` prop here? This might be helpful when we start building the screens.

Reply:
@smohamedjavid fixed

Reply:
@OmarBasem - Icon component doesn't accept `on-press`. We need to wrap it with a `pressable`. I'm not sure about the touch behaviour of the component whether the whole component is touchable or just the action area. Let's update it later 👍

Reply:
Thanks @smohamedjavid, fixed it